forceng.js

```diff
@@ -689,15 +692,19 @@ angular.module('forceng', [])
       return request(params);
 
     }
-    
+
     function getURLs() {
       return {proxyURL:proxyURL,oauthCallbackURL:oauthCallbackURL, useProxy: useProxy};
     }
 
+    function isOnMobile() {
+      return true;
```

Comment:
@jegauten RE: it being mobile only.

This line is what seemed to suggest it's going to be focussed on purely mobile. You're always returning true. Is this right?

Reply:
That's a good catch. I don't think we use that. Let me check with team.

Reply:
thanks Matt.

yes we dont use that we have a flag on our own in the app.js

Reply:
Can we safely remove it so it's clear it's not needed/used?

Reply:
yes it will be removed